docs/src/modules/ROOT/pages/optimization-algorithms/local-search.adoc

```diff
@@ -507,6 +507,50 @@ Use a lower tabu size than in a pure Tabu Search configuration.
   </localSearch>
 ----
 
+[#diversifiedLateAcceptance]
+== Diversified Late acceptance
+
```

Comment:
I'd add a note that it's a preview feature, with a link here:
https://docs.timefold.ai/timefold-solver/latest/upgrading-timefold-solver/backwards-compatibility#previewFeatures

Reply:
That section also needs updating with the addition of DLAS.